Merge back 'chore_release-8.5.0' into 'chore_release-pd-8.5.0'

ddcc4 opened the pull request:
This will also include @jbleon95's upcoming PR in `chore_release-8.5.0` once it's checked in.

sanni-t committed:
fix(shared-data): update lc display names (#18711)

Closes AUTH-2003

# Overview

Updates the display names to correct ones shown in design proposals.

## Risk assessment

None.

---------

Co-authored-by: Jethary <jethary@opentrons.com>

jbleon95 committed:
fix(api): fix chained transfers with keep_last_tip set to True and return_tip (#18715)

Fixes bugs related to tracking picked up or returned tips between liquid class transfers

jbleon95 reviewed (approved):
tip tracking bug fixes look good

sanni-t committed:
fix(api): error raising when there's no innergeometry for a well (#18728)

Closes RQA-4304

# Overview

The check that verifies that a submerge start location or retract end
location is not inside the liquid in well relies on knowing the height
of the liquid- either by calculating it based on liquid volume and well
inner geometry or by probing into the well.
We already skip the check if we don't know the liquid volume or haven't
probed the well yet, opting to simply log the decision instead. We
should also skip the check if a well's innergeometry isn't present
either otherwise the check will raise errors for any labware that
doesn't have inner geometry defined, like the `smc_384_read_plate`, or
if it's a custom labware.

So this PR makes it so that the check excepts on
`IncompleteLabwareDefinitionError` as well.

## Risk assessment

Low. Only skips a check that would have errored without being useful.

ddcc4 committed:
test(components): fix CommandText.test.tsx after we changed liquid class displayNames (#18738)

# Overview

This fixes a broken test after PR #18711, which changed the names of our
liquid classes (AUTH-2003).

Specifically, we renamed `Volatile` to `Volatile (80% ethanol)`.

## Test Plan and Hands on Testing

Ran test locally. Will also see if CI tests pass.

## Risk assessment
Low, test only.
